### DISCUSSION TITLE
Using Play CDN on Production site, is it safe?

### DISCUSSION THREAD

**Discussion by yabdab** [tailwindlabs/tailwindcss#7637] [Help]
I know the site reads "is not the **best** choice for production." , but can it be safe to use on low traffic sites?

What are the cons to using the cdn on a production site?

**Comment by adamwathan** [comment]
It's safe in a security sense if that's what you mean, it's just not a good choice for performance.

The JS script is a lot bigger than your CSS file would be, and the styles are applied at run-time which means you can have flashes of unstyled content. This is especially noticeable if you ever have elements that are dynamically added to the page after it first loads.

**Reply by yabdab** [reply]
Is there a way to build a css file that contains most of what TW offers? 
I am working on project that builds content dynamically and using conventional jit build methods are not possible.

**Reply by jtheisen** [reply]
@adamwathan As to the flickers, all I would need is a callback or event from PlayCDN when it's done with the styles. That way, I could hide new content until that moment.

There are many scenarios where nodejs-based builds are not an option. I have some ASP.NET-based projects, for example.
